hoot-core/src/main/cpp/hoot/core/conflate/extractors/FeatureExtractor.h

```diff
@@ -43,6 +43,8 @@
 namespace hoot
 {
 
+class OsmMap;
+
```

Comment:
Here a new forward declaration was needed because it relied on a duplicate forward declaration that I removed.